xrootd/xrdproto/protocol.go

```diff
@@ -116,3 +118,62 @@ const (
 	// DataServer indicates whether this is a data server.
 	DataServer
 )
+
+type FilepathRequest interface {
```

Comment:
please document this interface and its methods.

Reply:
Just an additional clarification: this interface will be implemented by all requests that use paths: `open`, `mv`, `rm`, `rmdir`, etc...
So if a redirect occurs as a response to the one of these request, an opaque data will be added to that request.